packages/explorer/src/css/home.module.css

```diff
@@ -0,0 +1,9 @@
+body {
+  font-family: Cabin, sans-serif;
+  background-color: #e8eaf6;
+}
```

Comment:
We'll also want a CSS reset if we're to ensure it looks the same across all browsers and devices. Though perhaps MUI already implements this for us?